samples/CosmosDbIntegtation/README.md

```diff
@@ -0,0 +1,65 @@
+    * Limitations: Write-around caching does have its own trade-offs, If data is accessed frequently right after being written, read latency may suffer due to the data not being initially cached. Additionally, there is still a possibility of increased read latency for less frequently accessed data, as it might not be cached at all.
+    ![Image](Models/WriteAround.png)
+
+1. ReadThrough: Read-through caching is a data caching technique that aims to improve read performance by reducing the latency associated with fetching data from the database. In this caching pattern, when a read request is made, the cache is the first point of access to check for the requested data. If the data is not found in the cache (a cache miss), the caching system automatically retrieves the data from the database and caches it for future use, which makes it accessible for the requesting application. 
+    * Benefits: Read-through caching offers improved read latency by serving frequently accessed data directly from the cache, reducing retrieval time from the slower underlying storage. This enhances application responsiveness and is particularly beneficial for read-intensive workloads. Moreover, read-through caching simplifies application code by automatically handling data retrieval and caching, freeing developers from managing these complexities. The cache adapts to changing access patterns over time, continually optimizing read performance. Additionally, read-through caching helps distribute read traffic, alleviating the load on the underlying storage and contributing to a more efficient system. Overall, it is a valuable strategy for data-intensive applications that rely heavily on read operations. 
+    * Limitations: Read-through caching may be inefficient for write-heavy workloads, leading to higher write latency. Limited cache size can result in frequent evictions, affecting less frequently accessed data. Data staleness issues may occur when frequent data changes are not immediately reflected in the cache. Using the read-through caching strategy requires careful consideration for invalidation strategies and eviction policies.
+   ![Image](Models/ReadThrough.png)
+
+## Description of each Redis Trigger
+1. PubSub: PubSub is a way for different programs to communicate with each other using messages. A publisher can send a message to a channel, and any number of subscribers listening to that channel can receive it. Pub/Sub uses a fire and forget message delivery system meaning that a message will be delivered once if at all. So, if a subscriber misses a message due to network error or some other issue, it will never receive it. You can read more about pubsub on the Redis docs
+    a.	Write Through, Write Behind: Every time a key is set or changed, the information is stored in CosmosDB. The same is true for messages sent on a specified Pubsub channel.
```

Comment:
These nested lists should still use numbers I believe. Otherwise they're all in one line (check the rich preview of this file)